core/domain/collection_services.py

```diff
@@ -290,11 +292,14 @@ def get_learner_collection_dict_by_id(
                 summary.exploration_model_last_updated
             ) if summary else None,
             'thumbnail_icon_url': utils.get_thumbnail_icon_url_for_category(
-                summary.category),
+                summary.category) if summary else
```

Comment:
I have reverted this. The issue that I found was that summary was a falsy object. Now for all other keys on the function before accessing a given summary's property value (e.g id, title, category etc) you first checked to see if summary is a truthy object. This was not done for thumbanil_icon_url and thumbnail_bg_color.